File: polocloud-components/component-terminal/src/main/java/dev/httpmarco/polocloud/component/terminal/command/arguments/StringArrayArgument.java
package dev.httpmarco.polocloud.component.terminal.command.arguments;

import dev.httpmarco.polocloud.component.terminal.command.CommandArgument;

public final class StringArrayArgument extends CommandArgument<String> {

    public StringArrayArgument(String key) {
        super(key);
    }

    @Override
    public String buildResult(String input) {
        return input;
    }
}

File: agent/src/main/kotlin/dev/httpmarco/polocloud/agent/runtime/local/terminal/arguments/type/StringArrayArgument.kt
package dev.httpmarco.polocloud.agent.runtime.local.terminal.arguments.type

import dev.httpmarco.polocloud.agent.runtime.local.terminal.arguments.TerminalArgument
import dev.httpmarco.polocloud.agent.runtime.local.terminal.arguments.InputContext

class StringArrayArgument(key: String) : TerminalArgument<String>(key) {

    override fun buildResult(input: String, context: InputContext): String {
        return input
    }
}